### PR TITLE
Add link to Trino code-style

### PR DESCRIPTION
**Pull request by hashhar**

### CODE REVIEW

File: development/index.md [hunk 1]
```diff
@@ -41,7 +41,8 @@ members of the community, and spend a significant amount of time reviewing commu
 
 ### Code style
 
-The Trino code base has a consistent style throughout.  The style is documented for Java code in the
+The Trino code base has a consistent style throughout which is documented
+[here](https://github.com/trinodb/trino/blob/master/.github/DEVELOPMENT.md#code-style).  The style is documented for Java code in the
```
[the remark this answers is not in the document]
Comment (mosabua) [reply development/index.md:45]: Also wrap this paragraph

**Review by mosabua — APPROVED** [review]
Good to go after the minor changes are done

File: development/index.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (mosabua) [reply development/index.md]: Fixed

**Comment by hashhar** [comment]
Sorry for the delay. Thanks a lot for finishing it up @mosabua.

**Pull request merged**